test/unit/karma.local.conf.js

```diff
@@ -18,6 +18,7 @@ module.exports = function (config) {
     },
     webpack: {
       ...karmaBaseConf.webpack,
+      optimization: { runtimeChunk: false },
```

Comment:
Why are we only doing it for local unit tests?

Reply:
I don't think we run multiple times consecutively for cicd. This only speed it up when we want to make changes and rerun quickly. [BrowserStack unit tests](https://gitlab.ddbuild.io/DataDog/browser-sdk/-/jobs/663886729) would fail without `splitChunk:false`.